### PR TITLE
[MIG] account_move_batch_validate: Migration to 9.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis. There's no extra commits to rescue from 8.0? Check migration guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0

File: account_move_batch_validate/__openerp__.py [hunk 1]
```diff
@@ -1,45 +1,22 @@
...
     'author': "Camptocamp,Odoo Community Association (OCA)",
     'category': 'Finance',
     'complexity': 'normal',
     'depends': [
-        'account',
-        'account_default_draft_move',
+        'account_accountant',
         'connector',
     ],
     'website': 'http://www.camptocamp.com',
```
Comment (simahawk) [inline account_move_batch_validate/__openerp__.py]: https://github.com/OCA/account-financial-tools

File: account_move_batch_validate/models/account.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    _logger.debug('Can not `import connector`.')
+    import functools
+
+    def empty_decorator_factory(*argv, **kwargs):
+        return functools.partial
+    job = empty_decorator_factory
+
+
+class AccountMove(models.Model):
+
```
Comment (simahawk) [inline account_move_batch_validate/models/account.py]: drop this empty line

File: account_move_batch_validate/test/test_batch_validate.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+                'done',
+                'Job is in state {0}, should be done'.format(job.state)
+            )
+        else:
+            job.perform(session)
+
+        return job
+
+    def test_batch_validate(self):
+
```
Comment (simahawk) [inline account_move_batch_validate/test/test_batch_validate.py:42]: can you skip some empty lines?

**Review by simahawk — CHANGES_REQUESTED** [review]
code review only: LGTM. Just a bunch of more or less nitpicking remarks.

**Comment by simahawk** [comment]
@oleksandrpaziuk pls, check Pedro's comment about history.

**Comment by ghost** [comment]
There are only 5 `OCA Transbot updated translations from Transifex` commits. I don't sink we need them for history.

**Comment by pedrobaeza** [comment]
Yes, we need them because they are translations, but you have to squash them together. What we don't want is that detailed history. Please follow https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0 and https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests guides

File: account_move_batch_validate/models/account.py [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        """Unmark moves for delayed posting, and cancel the jobs."""
+        self.write({'to_post': False})
+        self._cancel_jobs()
+
+
+@job(default_channel='root.account_move_batch_validate')
+def validate_one_move(session, model_name, move_id):
+    """Validate a move, and leave the job reference in place."""
+    move_ids = session.env[model_name].browse([move_id])
+    if move_ids.exists():
```
Comment (guewen) [inline account_move_batch_validate/models/account.py:115]: Code could be a bit simpler IMO: here we could check if `to_post` is still True, if not just leave with returning a message saying the move is not to post anymore. Thus we can remove the `unmark_for_posting` and `_cancel_jobs` methods. Anyway, this is a migration so maybe this PR is not the right place to change.

**Review by guewen — DISMISSED** [review]

**Review by guewen — COMMENTED** [review]
No commits for the original authors?

**Comment by ghost** [comment]
@guewen Unfortunately, someone incorrectly moves the module from 8 to 9 (before me) and therefore commits did not remain in history. Just in case, check if you want.

**Pull request closed without merging**